package.json

```diff
@@ -4,28 +4,48 @@
   "description": "Demo app for React Native Elements UI Library",
   "author": "Monte Thakkar",
   "private": true,
-  "main": "node_modules/expo/AppEntry.js",
+  "main": "src/index.native.js",
   "dependencies": {
+    "@babel/polyfill": "^7.2.5",
     "@expo/vector-icons": "^9.0.0",
     "expo": "^32.0.0",
+    "expo-linear-gradient": "^3.0.0",
     "lodash": "^4.17.4",
+    "prop-types": "^15.7.2",
     "react": "16.5.0",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-31.0.0.tar.gz",
+    "react-art": "^16.8.3",
+    "react-dom": "^16.8.3",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz",
     "react-native-elements": "^1.1.0",
-    "react-native-ratings": "^5.3.0",
+    "react-native-ratings": "^6.3.0",
     "react-native-touchable-scale": "2.0.0",
-    "react-navigation": "1.5.8"
+    "react-native-vector-icons": "^6.3.0",
+    "react-native-web": "^0.10.0",
+    "react-navigation": "1.5.8",
```

Comment:
We can make this part of another PR, but we should update this to v3

Reply:
I believe that navigation v3 will work. See [docs](https://reactnavigation.org/docs/en/web-support.html)